### DISCUSSION TITLE
Compander option

### DISCUSSION THREAD

**Discussion by merc07** [egzumer/uv-k5-firmware-custom#429] [Ideas]
This works well for removing the background static/hiss even if the other radio does not have it enabled.
But using it this way makes the voice sound slightly off.
Could there be a way to tune this function to use it as a 'hiss removal' ?, maybe a level setting option ?

**Comment by egzumer** [comment]
Compander has its purpose well defined in the manual, it is not hiss removal and I will never attempt to use it for that. There are other settings in the RF chip like "AF TX LPF2 filter bandwidth" or "300 Hz AF response coefficient for RX" that may be adjusted, feel free to investigate them.

**Reply by merc07** [reply]
I had no idea how the compander would help with background, I read it on another forum where they said it made quite a difference.

I'll look into your suggestions, thanks for the quick response again...
